packages/common/src/web3Proofs.ts

```diff
@@ -103,7 +109,7 @@ function verifyRangeStatement(
             }
         };
 
-        if (properties.type === 'string' && properties.format === 'date-time') {
+        if (isTimeStamp(properties)) {
```

Comment:
I am unsure if we actually agreed to parse it is a date when `format = date-time` or if we decided to have a custom `type = timestamp` for this.

Reply:
Fixed this.